pkg/codegen/docs/gen.go

```diff
@@ -363,8 +383,8 @@ func resourceName(r *schema.Resource) string {
 	return strings.Title(tokenToName(r.Token))
 }
 
-func getLanguageDocHelper(lang string) codegen.DocLanguageHelper {
-	if h, ok := docHelpers[lang]; ok {
+func getLanguageDocHelper(dctx *docGenContext, lang string) codegen.DocLanguageHelper {
```

Comment:
Nit: I have a _slight_ preference for making this (and other functions that take a `*docGenContext` as their first param) methods on `*docGenContext`